src/test/suite/completions.test.ts

```diff
@@ -61,18 +62,33 @@ suite("New Didact URI completion provider tests", function () {
 
 async function testWeGetExpectedResult(textToInsert : string, expectedResult: string) {
 	const editor = await createTestEditor(testDocumentUri);
-	await vscode.commands.executeCommand('editor.action.selectAll');
-	await vscode.commands.executeCommand('type', {"text": textToInsert});
+	await delay(100);
+	await clearTextEditor(editor, textToInsert);
+	await delay(100);
+	await vscode.commands.executeCommand("cursorEnd");
+	await delay(100);
 	await vscode.commands.executeCommand("editor.action.triggerSuggest");
-	await delay(1000);
+	await delay(100);
 	await vscode.commands.executeCommand("acceptSelectedSuggestion");
+	await delay(100);
 	expect(editor.document.getText()).to.include(expectedResult);
 }
 
 function delay(ms: number) {
 	return new Promise( resolve => setTimeout(resolve, ms) );
 }
 
+async function clearTextEditor(textEditor: TextEditor, initializeWith = "") {
```

Comment:
the name of the function is a bit misleading. It is more an initializing than a clear.

Reply:
I think we misunderstood each other. I meant that 
`async function clearTextEditor(textEditor: TextEditor, initializeWith = "")`
is not really a clear but more a initializing because it doesn't really only clear the editor but also presets it with `initializeWith` content.